python/ray/serve/_private/controller.py

```diff
@@ -614,7 +614,7 @@ def shutdown(self):
             and proxy_state_is_shutdown
         ):
             logger.warning(
-                "All resources have shut down, shutting down controller!",
```

Comment:
@GeneDer reminder to mostly avoid exclamation points in log messages (and other technical writing)

Reply:
will do👍